md/bonita-layout.md

```diff
@@ -0,0 +1,20 @@
+# Bonita Layout
+
+::: info
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+This layout has different desktop and mobile versions.
+For the dektop version, the layout contains two rows.
+The first row has the logo, the name of the application on the left, as well as the user name, which, when clicked, opens the information about the current session. The last button of the first row opens a window that will let you switch between living applications.
+The second row contains the names of the pages.
+For the mobile version there is only one row that has the name of the application, as well as a button that opens a dropdown. The dropdown will contain the names of the pages, the user name which, when clicked opens the information about the current session, as well as the button that opens the navigation between applications.
+
+## Customizing this layout
+
+You can customize this layout by following the [customize-resource-layouts](customize-resource-layouts.md) steps.
+
+## Adding a logo to the layout
+
+The first option to change the theme that is used by the application is to add a "logo.png" file in the "resources/images/" folder. If the folder does not exist, create it manually.
+The second option is adding an image asset to the layout in UI Designer, and using the asset name.
```

Comment:
in the UI Designer, and using this asset directly in the URL property of the image widget.

Reply:
done